src/python/pants/jvm/target_types.py

```diff
@@ -168,6 +168,19 @@ class JvmProvidesTypesField(StringSequenceField):
     )
 
 
+class JvmArtifactExcludeDependenciesField(StringSequenceField):
+    alias = "exclude_dependencies"
```

Comment:
This equivalent field in Pants v1 was called `excludes`. `exclude_dependencies` seems wordy. Any reason to not use the same name as from v1?